x-pack/plugins/security/public/hacks/on_session_timeout.js

```diff
@@ -65,7 +65,9 @@ module.config(($httpProvider) => {
 
     function interceptorFactory(responseHandler) {
       return function interceptor(response) {
-        if (!isUnauthenticated && !isSystemApiRequest(response.config) && sessionTimeout !== null) {
+        // If another interceptor has rejected a request, then config will be undefined.
+        const isSystemRequest = response.config ? isSystemApiRequest(response.config) : false;
```

Comment:
Without this  condition, a blocked request to `api/ui_metric` will throw an error inside of `isSystemApiRequest` when it tries to access a property of `config`.